SConstruct

```diff
@@ -1,34 +1,36 @@
 # Preliminaries
 import os
+import sys
+sys.dont_write_bytecode = True # Don't write .pyc files
 
 # Test for proper prerequisites and setup
-from setup import setup_test
-[user_configs, mode, sf, cache_dir] = setup_test(ARGUMENTS)
-import gslab_scons 
+from configuration_test import configuration_test
+[mode, sf, cache_dir] = configuration_test(ARGUMENTS, 
+                                           gslab_python_version = '4.0.0')
+import gslab_scons as gs
+import gslab_scons.log as log
 import yaml
 import atexit
-import gslab_scons.log as log
 
-# Start log
-mode = ARGUMENTS.get('mode', 'develop') # Gets mode; defaults to 'develop'
-vers = ARGUMENTS.get('version', '') # Gets release version; defaults to ''
+# Start log after getting mode and release version
+mode = ARGUMENTS.get('mode', 'develop') 
+vers = ARGUMENTS.get('version', '') 
 log.start_log(mode, vers)
 
-# Defines environment
+# Define the SCons environment
 env = Environment(ENV = {'PATH' : os.environ['PATH']}, 
                   IMPLICIT_COMMAND_DEPENDENCIES = 0,
-                  BUILDERS = {'Tablefill'   : Builder(action = gslab_scons.build_tables),
-                              'BuildLyx'    : Builder(action = gslab_scons.build_lyx),
-                              'BuildR'      : Builder(action = gslab_scons.build_r),
-                              'BuildStata'  : Builder(action = gslab_scons.build_stata),
-                              'BuildPython' : Builder(action = gslab_scons.build_python)},
+                  BUILDERS = {'Tablefill':   Builder(action = gs.build_tables),
+                              'BuildLyx':    Builder(action = gs.build_lyx),
+                              'BuildStata':  Builder(action = gs.build_stata),
+                              'BuildPython': Builder(action = gs.build_python)},
                   user_flavor = sf)
```

Comment:
I don't love the fact that we refer to the stata_flavor by four names
* stata_flavo(u)r
* user_flavor
* stata_executable
* sf

I'd like to see this standardized.